src/Raven.Studio/typescript/components/pages/database/documents/allRevisions/AllRevisions.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import { rtlRender } from "test/rtlTestUtils";
+import { composeStories } from "@storybook/react";
+import * as Stories from "./AllRevisions.stories";
+
+const { AllRevisionsStory } = composeStories(Stories);
+
+describe("AllRevisions", () => {
+    beforeAll(() => {
+        Object.defineProperty(HTMLElement.prototype, "scrollWidth", {
```

Comment:
move to common place + talk with @M4xymm  about that.

Reply:
It's moved here https://github.com/ravendb/ravendb/pull/19749